adoc/chapters/programming_interface.adoc

```diff
@@ -1433,12 +1441,19 @@ info::context::atomic_fence_order_capabilities
 ----
 
     @ [.code]#std::vector<memory_order>#
-   a@ Returns the set of memory orderings supported by [code]#atomic_fence#
-      on all devices in the context, which is guaranteed to include
-      [code]#relaxed#, [code]#acquire#, [code]#release# and [code]#acq_rel#.
+   a@ This query applies only to the capabilities of [code]#atomic_fence# when
+      applied to memory that can be concurrently accessed by multiple devices
+      in the context.  If these capabilities are not uniform across all devices
+      in the context, the query reports only the capabilities that are common
+      for all devices.
+
+Returns the set of memory orders supported by these [code]#atomic_fence#
+operations.  When a context returns a "stronger" memory order in this set, it
+must also return all "weaker" memory orders.
 
-The memory ordering of the context determines the behavior of fence operations applied
-to any memory that can be concurrently accessed by multiple devices in the context.
+At a minimum, each context must support [code]#memory_order::relaxed#,
+[code]#memory_order::acquire#, [code]#memory_order::release#, and
+[code]#memory_order::acq_rel#.
```

Comment:
All these? A fence with a relaxed memory order might be useless. But can we imagine a device specialized to produce data while another one could only consume?

Reply:
This PR does not add any new requirements here.  Even the old wording required all of these memory orderings.